src/object.d

```diff
@@ -782,6 +782,21 @@ class TypeInfo_Function : TypeInfo
     string deco;
 }
 
+unittest
+{
+    class C
+    {
+       void func() {}
+       void func(int a) {}
+       int func(int a, int b) { return 0; }
+    }
```

Comment:
I made the class `C ` abstract and only declared the functions as you suggested. Is this better?